lib/src/base-styles/_nx-page-header.scss

```diff
@@ -21,6 +21,7 @@ $main-header-product-version-row-height: 1fr;
   border-bottom: 1px solid $nx-header-border;
   flex-shrink: 0; // needed for IE11
   height: $main-header-height;
+  padding-left: $nx-spacing-md;
```

Comment:
> I feel like this would make more sense on `nx-page-header__inner`. As is here, the right and left edges of `.nx-page-header__inner` do not line up with the left and right edges of `.nx-page-content`

Yes: https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/29346959cfaa385b15845b24b4c5136c6aa0dedd